rule-manager/rule-manager-client/package.json

```diff
@@ -12,26 +12,33 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@babel/core": "^7.12.9",
-    "@babel/preset-env": "^7.12.7",
-    "@babel/preset-react": "^7.12.7",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "@emotion/babel-preset-css-prop": "^11.0.0",
+    "@types/react": "^16.14.2",
+    "@types/react-dom": "^16.9.10",
     "babel-loader": "^8.2.2",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^5.0.1",
     "html-webpack-plugin": "^4.5.0",
     "style-loader": "^2.0.0",
-    "ts-loader": "^8.0.11",
+    "ts-loader": "^8.0.12",
     "tsconfig-paths-webpack-plugin": "^3.3.0",
-    "typescript": "^4.1.2",
-    "webpack": "^5.10.0",
+    "typescript": "^4.1.3",
+    "webpack": "^4.44.2",
```

Comment:
Presume this downgrade is necessary, but it'd good to document why!

Reply:
It's required for `webpack-dev-middleware`, which is a dep of `webpack-dev-server`, but need to check to see if using v5 breaks anything, will investigate further.

Reply:
I upgraded to webpack 5 again, and removed `webpack-dev-server` for now, as it appears to be unused.